libgdx/libgdx
# Catch the cursor on Android

Have you ever wanted to play a first-person game using a mouse on Android? Yeah, me neither.

Test procedure:

* Connect a mouse and keyboard to an Android device on Android 8 or newer. I achieved this with an OTG cable and USB hub.
* Launch `InputTest`.
* See if everything seems sane before and after hitting `G`. Also after the app loses focus.

Obviously would be better to test with a real game, but it is what it is.

Resolves #6677.
